test/WebJobs.Script.Tests/WebScriptHostHandlerTests.cs

```diff
@@ -57,6 +58,17 @@ public WebScriptHostHandlerTests()
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
         }
 
+        [Fact]
+        public async Task SendAsync_HostInErrorState_Returns503()
+        {
+            _managerMock.SetupGet(p => p.IsRunning).Returns(false);
```

Comment:
Yes. Currently, this doesn't test the newly introduced behavior and would pass even before the changes were made. We need a test that covers the scenario and passes with the fix in place (i.e. a test that would fail without your change)